packages/flutter/lib/src/widgets/icon.dart

```diff
@@ -83,6 +84,11 @@ class Icon extends StatelessWidget {
   /// ```
   final Color color;
 
+  /// Semantic label for the icon.
+  ///
+  /// See [Semantics.label];
```

Comment:
This should probably document that this label will be read out if the app is navigated with a11y technologies (e.g. TalkBack or VoiceOver).